apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.component.ts

```diff
@@ -149,8 +150,15 @@ export class AssetProfileDialog implements OnDestroy, OnInit {
           scraperConfiguration: JSON.stringify(
             this.assetProfile?.scraperConfiguration ?? {}
           ),
-          symbolMapping: JSON.stringify(this.assetProfile?.symbolMapping ?? {})
+          symbolMapping: JSON.stringify(this.assetProfile?.symbolMapping ?? {}),
+          countries: JSON.stringify(
+            assetProfile.countries.map((e) => {
```

Comment:
`assetProfile.countries.map(({ code, weight }) => {`

Reply:
Why the mapping though? 🤔

Reply:
Map so we only show the data the user needs to input